server/ai_process.go

```diff
@@ -108,7 +108,9 @@ type liveRequestParams struct {
 	outSegmentTimeout      time.Duration
 
 	// Stops the pipeline with an error. Also kicks the input
-	stopPipeline func(error)
+	kickInput func(error)
+	// Cancels the execution for the given Orchestrator session
+	cancel context.CancelFunc
```

Comment:
What's the difference between this and `stopProcessing`? Maybe this could be renamed to `kickOrch` and `stopProcessing` to `kickOrchWithErr` or sth like that?

Reply:
Good idea. Renamed to kichOrch. I think stopProcessing() is ok.